electrical/calibration.MD

```diff
@@ -0,0 +1,183 @@
+
+![](../images/calibration/is11.png)
+**Figure 11**
+
+	(i) To verify that the corner steering motors have been calibrated, move the M1 and M2 sliders to their middle positions (halfway between min and max) and verify that the wheels all point exactly straight forwards. If they do not point straight forwards, verify that your min and max values are correct and ensure that no signal clipping is happening on the encoders.
+
+	(j) After the motors have both been tuned go to General settings and set the Motor current Max to be 3.0 Amps for each motor.
+	(k) Repeat Step 9 for the remaining corner motors
+
+## 2. Manual Calibration
+This section is to preform a manual tuning and calibration of the Corner motor system. This will be using the Ziegler-Nichols tuning method. For a more in depth explaination of how this process works you can visit:
+
+* [Tuning for PID Controllers](http://faculty.mercer.edu/jenkins_he/documents/TuningforPIDControllers.pdf)
+
+These steps assume you have already set the max/min values of the encoders in the position tab, as well as verified the encoder direction. If you have not done those follow the steps in the Corner Calibration
+
+1. Determine the QPPS of your system. This is the quadrature pulses per second at max speed of the motor. For that run your motor at max speed using the PWM settings tab, and look at the motor speed. This value just has to be initially close, so generally you can use the same number for all of the same motor types ie. Drive vs Corner motors.
+2. In the GUI under the position settings tab have it graph the Motor(x) position, and Motor(x) setpoint, where(x) is the motor you are tuning.
+3. Go to the Position Tab. Set all PID gains to be zero.
+4. Enter in the QPPS from step 1.
+5. Increase the P value and then use the position slider to command the setpoint of the motor. Initially I started my P value at 10 for the first increment. The motor should move to this position. If it stops immediately at the value then the system is over- damped. Keep increasing the P value until the motor beings to have a stead-state osciallation which doe NOT damp out to the setpoint. This value is called Kc. I found mine to be usually between 10-20 for values of P, which I incrimented in steps of 0.5 at a time.
+6. Now that the system is oscillating back and forth you have to get the period of the oscillation. Do this by starting a timer and timing how long it takes to do 30 os- ciallations. For mine it was around 10 seconds. Divide that time by the number of oscillations, this gives you the period, which is called Pcr. I found this to be generally around 0.3s
+7. The approximate gains can be found using the following equations:
+P = 0.6Kc (1)
```

Comment:
these equations don't render well

i would use a box, like from https://stackoverflow.com/a/58758521/4292910

Reply:
I moved the equations to separate lines. That seems to work. The equation numbers don't render well though so I commented them out.